.github/workflows/lint_python.yml

```
@@ -6,10 +6,9 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v3
-      - run: |
-           sudo apt-get -q update
-           sudo apt-get -qq upgrade
-           sudo apt-get -qq install curl ffmpeg libcurl4-openssl-dev libssl-dev motion ssh v4l-utils
+      - run: sudo apt-get -q update
+      - run: sudo apt-get -qq upgrade
+      - run: sudo apt-get -qq install curl ffmpeg libcurl4-openssl-dev libssl-dev motion ssh v4l-utils
```

What do we need `ssh` for, actually? Also `curl`, `ffmpeg`, `motion` and `v4l-utils` are not required for the build, i.e.:
```suggestion
      - run: sudo apt-get -qq install libcurl4-openssl-dev libssl-dev
```

---

Let's slim down our dependencies only **after** we get #2325 fixed.